### PR TITLE
fix: `useSyncExternalStoreWithTracked` equalityFn should return truthy when there are no tracked keys

### PR DESCRIPTION
**Pull request by jxom**
## Description

_Concise description of proposed changes_

## Additional Information

- [ ] I read the [contributing docs](/tmm/wagmi/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

File: packages/react/src/hooks/utils/useSyncExternalStoreWithTracked.test.tsx [hunk 1]
```diff
@@ -104,11 +124,33 @@ describe('useSyncExternalStoreWithTracked', () => {
...
+        {
+          "baz": "wagmi",
+          "foo": "bar",
+          "isGonnaMakeIt": true,
+        },
       ]
     `)
+  })
+
+  it('does not rerender when no values are being tracked', async () => {
```
Comment (jxom) [inline packages/react/src/hooks/utils/useSyncExternalStoreWithTracked.test.tsx:136]: this is the new test being added

File: packages/react/src/hooks/accounts/useDisconnect.test.ts [hunk 2]
```diff
@@ -3,13 +3,18 @@ import { useAccount } from './useAccount'
 import { useConnect } from './useConnect'
 import { UseDisconnectConfig, useDisconnect } from './useDisconnect'
 
+function useTestAccount() {
+  const { ...values } = useAccount()
+  return values
+}
```
Comment (tmm) [inline packages/react/src/hooks/accounts/useDisconnect.test.ts:9]: Want to add `useTestAccount` to the `test/utils.ts` and export from `test/index.ts`?

File: packages/react/src/hooks/accounts/useNetwork.test.ts [hunk 3]
```diff
@@ -16,6 +16,11 @@ import {
   useSwitchNetwork,
 } from './useSwitchNetwork'
 
+function useTestNetwork() {
+  const { ...values } = useNetwork()
+  return values
+}
```
Comment (tmm) [inline packages/react/src/hooks/accounts/useNetwork.test.ts:22]: Could add this to `test/utils.ts` too!

**Review by tmm — APPROVED** [review]

File: packages/react/src/hooks/accounts/useDisconnect.test.ts (hunk 2 above)
Comment (jxom) [reply packages/react/src/hooks/accounts/useDisconnect.test.ts:9]: sg!

**Pull request merged**